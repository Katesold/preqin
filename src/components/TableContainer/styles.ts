import styled from "styled-components";

export const StyledTableContainer = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
  padding: 20px;

  h1 {
    text-align: center;
    padding-bottom: 20px;
  }
`;
