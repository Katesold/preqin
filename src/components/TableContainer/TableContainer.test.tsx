import { render, screen } from "@testing-library/react";

import { TableContainer } from "./TableContainer";

window.alert = jest.fn();

test("renders table container and fetches data", async () => {
  const { container } = render(<TableContainer />);

  expect(container).toMatchSnapshot();
  expect(screen.getByText("Firm ID")).toBeInTheDocument();
});
