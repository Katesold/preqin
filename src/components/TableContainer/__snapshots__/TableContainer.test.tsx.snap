// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders table container and fetches data 1`] = `
<div>
  <div
    class="sc-fsYfdN gcAEzM"
  >
    <h1>
      Investors List
    </h1>
    <table
      class="sc-guDLey bZjJOl"
    >
      <thead
        class="sc-dmyCSP ksQsTG"
      >
        <tr
          class="sc-jTQCzO kKsiDq"
        >
          <th
            class="sc-eDLKkx dfMviv"
          >
            Firm ID
          </th>
          <th
            class="sc-eDLKkx dfMviv"
          >
            Firm Name
          </th>
          <th
            class="sc-eDLKkx dfMviv"
          >
            Type
          </th>
          <th
            class="sc-eDLKkx dfMviv"
          >
            Date Added
          </th>
          <th
            class="sc-eDLKkx dfMviv"
          >
            Address
          </th>
          <th
            class="sc-eDLKkx dfMviv"
          >
            Action
          </th>
        </tr>
      </thead>
      <tbody
        class="sc-hLQSwg gYvGVw"
      />
    </table>
  </div>
</div>
`;
